### PR TITLE
Added includeSuiteFailure config option

### PR DESCRIPTION
**Pull request by 4eb0da**
Sometimes suite is failed completely, for instance, because of SyntaxError
Right now `jest-html-reporter` says nothing about this suites, suite is represented only in counter of failed suites
To figure out what is happening one should use other reporter, for example, default
This pr adds option to display info about this suites and failureMessage from them

### CODE REVIEW

**Comment by 4eb0da** [comment]
@Hargne take a look, pls

File: src/htmlreporter.ts [hunk 1]
```diff
@@ -123,6 +123,58 @@ class HTMLReporter {
...
+      {
+        class: `suite-time${
+          executionTime2 >
+          (this.getConfigValue("executionTimeWarningThreshold") as number)
+            ? " warn"
+            : ""
+        }`,
+      },
+      `${executionTime2}s`
+    );
```
Comment (Hargne) [inline src/htmlreporter.ts:143]: Any reason to why the constant names are ending with 2?

Comment (4eb0da) [reply src/htmlreporter.ts]: Oops
Forgot to change them after refactor
Fixed, thx!

Comment (4eb0da) [reply src/htmlreporter.ts]: @Hargne

**Review by Hargne — APPROVED** [review]

**Pull request merged**